src/ziggurat/init.clj

```diff
@@ -88,18 +93,36 @@
   (mount/stop #'server/server)
   (stop-messaging))
 
+(defn start-batch-consumer [args]
+  (-> (mount/only #{#'executor-service/thread-pool
+                    #'consumer-driver/consumer-groups})
+      (mount/with-args (:batch-routes args))
+      (mount/start)))
+
+(defn stop-batch-consumer []
```

Comment:
Should we not stop the consumer group first? Will the consumer group not stop automatically when thread pool is stopped? Let me know if I understand this correctly.

Reply:
`mount/only` only specifies the states while ordering of `mount/stop` is based on dependencies between states.